src/ai/backend/agent/server.py

```diff
@@ -59,15 +59,18 @@
 from ai.backend.common.dto.agent.response import AbstractAgentResp, PurgeImagesResp
 from ai.backend.common.dto.manager.rpc_request import PurgeImagesReq
 from ai.backend.common.etcd import AsyncEtcd, ConfigScopes
-from ai.backend.common.events.image import (
+from ai.backend.common.events.event_types.image import (
     ImagePullFailedEvent,
     ImagePullFinishedEvent,
     ImagePullStartedEvent,
 )
-from ai.backend.common.events.kernel import (
-    KernelLifecycleEventReason,
+from ai.backend.common.events.event_types.kernel.anycast import (
     KernelTerminatedEvent,
 )
```

Comment:
Some are separated into `anycast` and `broadcast`, and some are all included in the upper-level package, so please unify the package structure consistently.